packages/core/integration-tests/test/server.js

```diff
@@ -118,17 +119,22 @@ describe('server', function() {
 
     await b.run();
 
-    b.reporterRunner.report({
```

Comment:
Since this is no longer directly exposed, we have to test this in a more end-to-end way without using internals, which is probably for the better.